pygeoapi/openapi.py

```diff
@@ -652,18 +655,38 @@ def get_oas_30(cfg):
                     'summary': 'Get a {} tile'.format(v['title']),
                     'description': v['description'],
                     'tags': [k],
-                    'operationId': 'describe{}Tiles'.format(k.capitalize()),
-                    'parameters': [
-                        items_f,
-                    ],
+                    'operationId': 'get{}Tiles'.format(k.capitalize()),
+                    'parameters': [{
+                        'name': 'f',
+                        'in': 'query',
+                        'description': 'The optional f parameter indicates the output format which the server shall provide as part of the response document.',  # noqa
+                        'required': False,
+                        'schema': {
+                            'type': 'string',
+                            'enum': [tp.format_type],
+                            'default': tp.format_type
+                        },
+                        'style': 'form',
+                        'explode': False
+                    }],
                     'responses': {
-                        '200': {'$ref': '#/components/responses/Tiles'},  # noqa # TODO: fix response format
                         '400': {'$ref': '{}#/components/responses/InvalidParameter'.format(OPENAPI_YAML['oapif'])},  # noqa
                         '404': {'$ref': '{}#/components/responses/NotFound'.format(OPENAPI_YAML['oapif'])},  # noqa
                         '500': {'$ref': '{}#/components/responses/ServerError'.format(OPENAPI_YAML['oapif'])}  # noqa
                     }
                 }
             }
+            mimetype = tile_extension['format']['mimetype']
+            paths[tiles_data_path]['get']['responses']['200'] = {
```

Comment:
This response is for the path `/collections/{collectionId}/tiles/{tileMatrixSetId}/{tileMatrix}/{tileRow}/{tileCol}`